src/acorn/src/util.js

```diff
@@ -1,7 +1,3 @@
-export function isArray(obj) {
-  return Object.prototype.toString.call(obj) === "[object Array]"
-}
-
```

Comment:
Might not be necessary in a bit. I'll be opening a PR soon that starts the transition away from Acorn.